attestation-service/docs/restful-as.md

```diff
@@ -2,11 +2,54 @@
+```
+
+The value is a base64 encoded JWT. The body of the JWT is showed in the [example.token.json](./example.token.json).
+
+## Advanced Topic
```

Comment:
nit: `Topic` -> `Topics`. Could also potentially call this section `For Developers`

Reply:
Thanks. There is a section named `HTTPS Support` which seems more like a deployment configuration thus `Advanced Topics` would make more sense. But I am fine to change as I am not a native speaker to catch tiny difference : )

Reply:
Nah it's fine either way.